### PR TITLE
feat: Upgrade Mongo

### PR DESCRIPTION
**Pull request by ryanwohara**
# Description
Upgrading Mongo can be tricky but we found the golden path to get to v4.4.

# Reasons
The Mongo we ship is old and customers are asking how to update it. I have verified the path forward and written it down here.

[SERVER-2343](https://circleci.atlassian.net/browse/SERVER-2343)
[SERVER-2428](https://circleci.atlassian.net/browse/SERVER-2428)

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [X] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [X] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

[SERVER-2343]: https://circleci.atlassian.net/browse/SERVER-2343?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SERVER-2428]: https://circleci.atlassian.net/browse/SERVER-2428?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by kelvintaywl** [comment]
Ah I noted this is applicable for when internal MongoDB is used, and the base Docker image is https://hub.docker.com/r/bitnami/mongodb

Would this be correct?

**Comment by ryanwohara** [comment]
That is correct @kelvintaywl

**Review by atulsingh0 — APPROVED** [review]

**Comment by vjpandian** [comment]
@ryanwohara Don't have permission to merge this but this looks good to me. Thanks for the effort to publish this in our docs.

File: jekyll/_cci2/server/v4.1/operator/upgrading-mongo.adoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```yaml
+mongodb:
+  image:
+    tag: 4.0.27-debian-9-r118
+```
+
+And then `helm upgrade` your installation. 
+
+Once the `helm upgrade` has completed and Mongo has rolled, you will need to exec into the container (with the root password handy) to modify the compatibility version. (Be sure to replace `<password>` with your Mongo root password.)
+
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/server/v4.1/operator/upgrading-mongo.adoc:51]: exec into the `container` or `pod`? We have both on this page

[the remark this answers is not in the document]
Comment (ryanwohara) [reply jekyll/_cci2/server/v4.1/operator/upgrading-mongo.adoc:51]: Oooo good catch. I think as long as we're consistent it should be fine. Let's use `pod`.

**Review by rosieyohannan — APPROVED** [review]

**Comment by rosieyohannan** [comment]
@ryanwohara I just made a few changes for consistency and added the page to the sidebar. Approved so maybe check what I've done (let me know if any issues) and feel free to merge :-D

**Comment by ryanwohara** [comment]
Looks good! 🙇

**Pull request merged**